link/uprobe_multi_test.go

```diff
@@ -0,0 +1,133 @@
+package link
+
+import (
+	"errors"
+	"os/exec"
+	"testing"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/internal/testutils"
+	"github.com/cilium/ebpf/internal/unix"
+)
+
+func TestUprobeMulti(t *testing.T) {
+	testutils.SkipIfNotSupported(t, haveBPFLinkUprobeMulti())
+
+	prog := mustLoadProgram(t, ebpf.Kprobe, ebpf.AttachTraceUprobeMulti, "")
+
+	um, err := bashEx.UprobeMulti([]string{"bash_logout", "main"}, prog, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer um.Close()
+
+	testLink(t, um, prog)
+}
+
+func TestUprobeMultiInput(t *testing.T) {
+	testutils.SkipIfNotSupported(t, haveBPFLinkUprobeMulti())
+
+	prog := mustLoadProgram(t, ebpf.Kprobe, ebpf.AttachTraceUprobeMulti, "")
+
+	// One of Symbols or Addresses must be given.
+	_, err := bashEx.UprobeMulti([]string{}, prog, nil)
+	if !errors.Is(err, errInvalidInput) {
```

Comment:
Can you use `qt.Assert(t, qt.ErrorIs(...))`?

Reply:
ok, will change